code/modules/clothing/gloves/_gloves.dm

```diff
@@ -40,13 +40,20 @@
 	if(ismob(loc))
 		var/mob/M = loc
 		M.update_worn_gloves()
-
-/obj/item/clothing/gloves/wirecutter_act(mob/living/user, obj/item/I)
-	. = ..()
+		
+/obj/item/clothing/gloves/proc/can_cut_with(obj/item/I, mob/user)
```

Comment:
1. Rename `I` to something like `tool` as per new variable name guidelines
2. `user` is unused here, don't need it